src/visualization/parsers/settleLayout.ts

```diff
@@ -0,0 +1,267 @@
+/// <reference path="../../../node_modules/clarity-js/clarity.d.ts" />
+import { IParser } from "../components/Snapshot";
+import { IAttributes, ILayoutState, IElementLayoutState, IDoctypeLayoutState, ITextLayoutState, IIgnoreLayoutState, Action } from "clarity-js/clarity";
+import {ISettleElementLayoutState} from "./ISettleElementLayoutState";
+export default class Layout implements IParser {
```

Comment:
>Layout [](start = 21, length = 6)

99% of this is just a copy of the other layout right? Let's extend the class and just override whatever small bits are changing